x-pack/plugins/apm/public/services/rest/apm/transactions.ts

```diff
@@ -19,7 +19,7 @@ export async function loadTransaction({
   kuery
 }: IUrlParams) {
   try {
-    const result = await callApi<TransactionAPIResponse>({
+    const result = await callApi<TransactionWithErrorCount>({
```

Comment:
I tried to make a naming convention that the publicly available backend endpoints expose their interface with `APIResponse`. To follow this convention, can you name it `TransactionWithErrorCountAPIResponse`?

I'm open to other suggestions :)